README.md

```diff
@@ -1 +1,63 @@
-# bitcoin-association-switzerland
+# Bitcoin Association Switzerland (BAS) Repository
```

Comment:
```suggestion
# Bitcoin Association Switzerland (BAS)
```

As you pointed out, there is also the logo and a contract in this repo (the later might be removed at some time), so this title would make more sense.